### PR TITLE
Add link resource

### PR DESCRIPTION
**Pull request by mwsteb**

### CODE REVIEW

**Comment by mwsteb** [comment]
@azranel ready for review!

**Comment by azranel** [comment]
Could you merge master and add `belongs_to :user, Shareguru.User` (or smth like that) to Shareguru.Link resource? 😄

**Comment by azranel** [comment]
LGTM

**Comment by mwsteb** [comment]
@azranel  should be cool now

File: test/controllers/link_controller_test.exs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      email: "dude@example.com"
+    } |> Repo.insert
+    
+    {:ok, user: Repo.get(User, 123456) }
+  end
+
+  test "lists all entries on index", %{conn: conn, user: user} do
+    conn = conn
+    |> assign(:current_user, user)
+    |> get(link_path(conn, :index))
```
Comment (azranel) [inline test/controllers/link_controller_test.exs]: This assigning could be extracted to ConnCase 😄

**Review by azranel — CHANGES_REQUESTED** [review]

**Review by azranel — APPROVED** [review]

**Pull request merged**